server/models.py

```diff
@@ -12,7 +12,7 @@
 load_dotenv()
 
 HOST = os.getenv("FLASK_HOST")
-
+debug = True  

```

Comment:
debug should be False. Also, do we want to add an additional debug variable here? I think that your 403 response issue got solved so we don't need this.  @giancarlopernudisegura  should we consider adding a debug variable in the models.py for future reference?

Reply:
Don't know why I didn't get notified but we should just use the flask debug variable instead of creating a new one.